src/containers/App/App.js

```diff
@@ -135,251 +168,263 @@ export /* istanbul ignore next */ class App extends Component {
-                  <Route
-                    path={paths.clusterTriggerBindings.all()}
-                    exact
-                    component={ClusterTriggerBindings}
-                  />
-                  <Route
-                    path={paths.triggerTemplates.byName()}
-                    exact
-                    component={TriggerTemplate}
-                  />
-                  <Route
-                    path={paths.triggerTemplates.all()}
-                    exact
-                    component={TriggerTemplates}
-                  />
-                  <Route
-                    path={paths.triggerTemplates.byNamespace()}
-                    exact
-                    component={TriggerTemplates}
-                  />
-                  <Route
-                    path={paths.extensions.all()}
-                    exact
-                    component={Extensions}
-                  />
-                  {extensions
-                    .filter(extension => !extension.type)
-                    .map(({ displayName, name, source }) => (
-                      <Route
-                        key={name}
-                        path={paths.extensions.byName({ name })}
-                        render={({ match }) => (
-                          <Extension
-                            displayName={displayName}
-                            match={match}
-                            source={source}
-                          />
-                        )}
-                      />
-                    ))}
-
-                  <Route
-                    path={paths.kubernetesResources.all()}
-                    exact
-                    component={ResourceList}
-                  />
-                  <Route
-                    path={paths.kubernetesResources.byNamespace()}
-                    exact
-                    component={ResourceList}
-                  />
-                  <Route
-                    path={paths.kubernetesResources.byName()}
-                    exact
-                    component={CustomResourceDefinition}
-                  />
-                  <Route
-                    path={paths.kubernetesResources.cluster()}
-                    exact
-                    component={CustomResourceDefinition}
-                  />
-                  <Route
-                    path={paths.rawCRD.byNamespace()}
-                    exact
-                    component={CustomResourceDefinition}
-                  />
-                  <Route
-                    path={paths.rawCRD.cluster()}
-                    exact
-                    component={CustomResourceDefinition}
-                  />
-
-                  <Redirect to={urls.pipelineRuns.all()} />
-                </Switch>
-              </PageErrorBoundary>
-            </Content>
-          </>
-        </Router>
+        {loadingConfigError && (
+          <InlineNotification
+            kind="error"
+            title="Error getting data"
```

Comment:
will need to use `intl` for this message, also maybe 'Error loading configuration' for consistency with the loading state and to make it clearer what failed

Reply:
done